DICOMwebBrowser/DICOMwebBrowser.py

```diff
@@ -79,7 +79,7 @@ def setup(self):
     # Ensure that correct version of dicomweb-clien Python package is installed
     needRestart = False
     needInstall = False
-    minimumDicomwebClientVersion = "0.51"
+    minimumDicomwebClientVersion = "0.56.2"
```

Comment:
Please describe in the commit comment the rationale for increasing the minimum required version.

Reply:
Minimum version for `get_remaining` param is 0.55. Since we are updating, I just went to the latest version.

Reply:
OK, this is an acceptable explanation, please add it to the commit comment (and/or as a comment in the code above the `minimumDicomwebClientVersion =...` line).

Note that the DICOMweb-client package quite often has regressions (it broke several times during the past 1-2 years), so careful testing is needed with this new minimum version.